cmd/pilot-agent/main.go

```diff
@@ -68,6 +61,31 @@ var (
 			glog.V(2).Infof("version %s", version.Line())
 			glog.V(2).Infof("mesh configuration %#v", mesh)
 
+			// set values from registry platform
```

Comment:
this would also be clearer as a switch and per platform init, specially the line 78-88